web/thesauruses/clojure/1/operators.json

```diff
@@ -0,0 +1,223 @@
+      "code": "(let [x (volatile! 0)]\n  (doseq [addend (range 10)]\n    (vswap! x + addend))\n  @x)",
+      "name": "Addition and assignment operator"
+    },
+    "subtraction": {
+      "code": "(- 10 4)",
+      "name": "Subtraction operator"
+    },
+    "subtraction_assignment": {
+      "comment": "See note about the Addition and assignment operator above.",
+      "not-implemented": "true",
+      "name": "Subtraction and assignment operator"
+    },
+    "multiplication": {
+      "code": "(* 12 12)",
+      "name": "Multiplication operator"
+    },
+    "multiplication_assignment": {
+      "comment": "See note about the Addition and assignment operator above.",
+      "not-implemented": "true",
+      "name": "Multiplication and assignment operator"
+    },
+    "division": {
+      "code": "(/ 1024 16)",
+      "name": "Division operator"
+    },
+    "division_assignment": {
+      "comment": "See note about the Addition and assignment operator above.",
+      "not-implemented": "true",
+      "name": "Division and assignment operator"
+    },
+    "integer_division": {
+      "comment": "Clojure includes a Ratio type to represent rational ratios exactly, but this can be coerced to an int",
+      "code": "(int (/ 10 3))",
+      "name": "Integer division operator"
+    },
+    "integer_division_assignment": {
+      "not-implemented": "true",
+      "name": "Integer division and assignment operator"
+    },
+    "unary_plus": {
+      "comment": "All arithmetic operators can take arbitrarily many arguments, including `+`, but this provides unclear utility",
+      "code": "(+ 5)",
+      "name": "Unary plus operator"
+    },
+    "unary_minus": {
+      "comment": "All arithmetic operators can take arbitrarily many arguments, including `-`, but this provides unclear utility",
+      "code": "(- 10)",
+      "name": "Unary minus operator"
+    },
+    "modulus": {
+      "code": "(mod 18 8)",
+      "name": "Modulus (remainder) operator"
+    },
+    "modulus_assignment": {
+      "comment": "See note about the Addition and assignment operator above.",
+      "not-implemented": "true",
+      "name": "Modulus and assignment operator"
+    },
+    "increment": {
+      "comment": "See note about the Addition and assignment operator above. The `inc` function will increment the value and return the result, not overwrite any \"variable\".",
+      "code": "(inc 1)",
+      "name": "Increment (add 1) operator"
+    },
+    "decrement": {
+      "comment": "See note about the Addition and assignment operator above. The `dec` function will decrement the value and return the result, not overwrite any \"variable\".",
+      "code": "(dec 10)",
+      "name": "Decrement (subtract 1) operator"
+    },
+    "exponential": {
+      "comment": "No built in function, but easy enough to write.",
+      "code": "(defn exp [x n]\n  (reduce * (repeat n x)))",
```

Comment:
Same for these, don't write new functions, just report if the language presently can do it. (so "not-implemented")